docker/Dockerfile

```diff
@@ -41,10 +38,18 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --defau
 ENV PATH="/root/.cargo/bin:$PATH" \
     PGX_HOME=/usr/lib/postgresql/${PG_VERSION_MAJOR}
 
+# Copy Cargo.toml file
+WORKDIR /tmp
+COPY pg_search/Cargo.toml ./
+
```

Comment:
clever!  I was wondering how we'd *actually* do this in the Dockerfile.

Reply:
Maybe just expand the command to something like "Copy Cargo.toml because we need to extract the pgrx version"